satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -619,4 +619,26 @@ update offer (
 	where offer.expires_at >= offer.created_at
 )
 
-delete offer ( where offer.id = ? )
\ No newline at end of file
+delete offer ( where offer.id = ? )
+
+//--- user credit table ---//
+
+model user_credit (
+	key		user_id
+
+	field	user_id user.id	cascade
```

Comment:
If we delete this record when a user account is deleted, then we will also loose the record of who referred this user. I think we should keep this record

Reply:
see https://github.com/storj/storj/pull/2085#discussion_r290428198